### PR TITLE
TfLite Reverse missing datatype support

### PR DESCRIPTION
**Pull request by swatheesh-mcw**
-Adds bf16, f16 support for reverse
 -Adds bf16, f16 reverse unit tests

### CODE REVIEW

File: tensorflow/lite/kernels/reverse_test.cc [hunk 1]
```diff
@@ -12,12 +12,12 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```
Comment (rascani) [inline tensorflow/lite/kernels/reverse_test.cc:16]: These lines should go back to where they were.

File: tensorflow/lite/kernels/test_util.h [hunk 2]
```diff
@@ -15,6 +15,8 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_KERNELS_TEST_UTIL_H_
 #define TENSORFLOW_LITE_KERNELS_TEST_UTIL_H_
 
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```
Comment (rascani) [inline tensorflow/lite/kernels/test_util.h:19]: These lines should go back to where they were.

**Review by rascani — CHANGES_REQUESTED** [review]
LGTM, just a couple include moves to revert.

File: tensorflow/lite/kernels/reverse_test.cc (hunk 1 above)
Comment (swatheesh-mcw) [reply tensorflow/lite/kernels/reverse_test.cc]: Addressed the requested change in [#1b42f81](https://github.com/MCW-Dev/tensorflow/pull/31/commits/1b42f812cab737e1f4afff6d89506aa5c1aeb0bc)

File: tensorflow/lite/kernels/test_util.h (hunk 2 above)
Comment (swatheesh-mcw) [reply tensorflow/lite/kernels/test_util.h]: Addressed the requested change in [#1b42f81](https://github.com/MCW-Dev/tensorflow/pull/31/commits/1b42f812cab737e1f4afff6d89506aa5c1aeb0bc)

**Review by rascani — APPROVED** [review]

**Pull request merged**